simSPI/linear_simulator/utils.py

```diff
@@ -0,0 +1,136 @@
+"""contain miscellaneous function."""
```

Comment:
This file should not be merged. The content needs to be split in places where it belongs.